sections/03_categorize.md

```diff
@@ -238,12 +261,90 @@ Ratner, Bach, and Ré described labeled training data as "The _New_ New Oil"
 
+###### Data sharing is hampered by standardization and privacy considerations
+
+EHRs are designed and optimized primarily for patient care and billing purposes,
+meaning research is at most a tertiary priority. This presents significant
+challenges to EHR based research in general, and particularly to data intensive
+deep learning research. EHRs are used differently even within the same health
+care system [@pmid:24159271 @pmid:21347133]. Individual users have unique
+usage patterns, and different departments have different priorities which
+introduce missing data in a non-random fashion. Just et al. demonstrated that
+even the most basic task of matching patients can be challenging due to data
+entry issues [@pmid:27134610]. This is before considering challenges caused by
+system migrations and health care system expansions through acquisitions.
+Replication between hospital systems requires controlling for both these
+systematic biases as well as for population and demographic effects.
+Historically, rules-based algorithms have been popular in EHR-based research but
+because these are developed at a single institution and trained with a specific
+patient population they do not transfer easily to other populations
+[@doi:10.1136/amiajnl-2013-001935]. Wiley et al.
+[@doi:10.1142/9789813207813_0050] showed that warfarin dosing algorithms often
+under perform in African Americans, illustrating that some of these issues are
+unsolved even at a treatment best practices level. Lack of standardization
+makes it challenging for investigators skilled in deep learning to enter the
+field, as numerous data processing steps must be performed before algorithms are
+applied.
+
+Even if data were perfectly standardized, attempts to share data in this domain
+would still encounter technological and legal barriers. A responsibility to
+protect patient privacy limits the ability to  openly share large patient
+datasets. As described above, labeled data are already expensive to obtain. Even
+after they are generated restrictions on sharing can hamper their broad
+distribution. All of these factors combine to result in small samples sizes that
+restrict the number of parameters that can be trained in a model. Though the
+lack of sharing may also hamper reproducibility and physician confidence in
+results, recently described techniques such as Continuous Analysis
+[@doi:10.1038/nbt.3780] can allow reproducible computing on private data. Using
+such techniques intermediate results can be automatically tracked and shared
+without sharing the original data, which may help to address concerns around
+physician confidence.
+
+Raw data isn't the only point of concern in the domain with regards to privacy.
+Even without sharing data, algorithms trained on confidential patient data may
+present security risks or accidentally allow for the exposure of individual
+level patient data. Tramer et al. [@arxiv:1609.02943] showed the ability to
```

Comment:
I think we should also cite Shokri et al. (similar work, but for membership inference attacks) - [@arxiv:1610.05820]. I think this is important because it also points at why this is especially important for deep networks with many parameters. #329